visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -129,15 +137,28 @@ void MotionPlanningFrame::computePlanButtonClicked()
 void MotionPlanningFrame::computeExecuteButtonClicked()
 {
   if (move_group_ && current_plan_)
-    move_group_->execute(*current_plan_);
```

Comment:
Replacing `move_group_->execute()` with `move_group_->asyncMove()` is not possible, because behaviour is changed: While the former executes a previously planned, manually validated trajectory, the latter will actually plan a new trajectory and directly execute it - with no chance for inspection/validation.

**I cannot accept this PR in this form.** 

Looking deeper into the reasons, I found the following: There are essentially to methods to plan/execute:

- `move_group::plan()` and `move_group::move()` are calls to an _action_, that can be interrupted. The flag `plan_only` allows to switch off/on execution.
- However, `move_group::execute()` is a call to a service (`MoveGroupExecuteService`), which cannot be interrupted.

In order to allow stopping of a trajectory, the service should be turned into an action as well. As far as I can see, that changed could be hidden transparently behind the existing MoveGroupInterface. However, I'm not sure, whether some (3rd-party) code is directly calling the service. Within the MoveIt code base, I couldn't find any other usage.

@davetcoleman Do you think, this change is reasonable?

Alternatively, we do not allow to stop an executed-only trajectory.